src/routes/api/v1/user.js

```diff
@@ -2,16 +2,22 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { Router } from 'express'
+import { Router, json } from 'express'
 
 import { asyncMiddleware } from '../../../middleware/util.js'
 import { requireSessionUser } from '../../../middleware/auth.js'
 import { methodNotAllowed } from '../../../middleware/error.js'
 import { putBreachResolution, getBreaches } from '../../../controllers/breaches.js'
+import { addEmail, resendEmail, removeEmail, verifyEmail, updateCommunicationOptions } from '../../../controllers/settings.js'
```

Comment:
Suggestion (formatting): It’s getting a bit looong here. :)
```suggestion
import {
  addEmail,
  resendEmail,
  removeEmail,
  verifyEmail,
  updateCommunicationOptions
} from '../../../controllers/settings.js'
```